python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -428,24 +428,24 @@ def from_string(serialized: Optional[str]):
         )
 
 
-def _fetch_runs_not_in_backfill(
-    instance: DagsterInstance,
-    cursor: Optional[str],
-    limit: int,
-    filters: Optional[RunsFilter],
-) -> Sequence[RunRecord]:
-    """Fetches limit RunRecords that are not part of a backfill and were created before a given timestamp."""
-    runs = []
-    while len(runs) < limit:
-        # fetch runs in a loop and discard runs that are part of a backfill until we have
-        # limit runs to return or have reached the end of the runs table
-        new_runs = instance.get_run_records(limit=limit, cursor=cursor, filters=filters)
-        if len(new_runs) == 0:
-            return runs
-        cursor = new_runs[-1].dagster_run.run_id
-        runs.extend([run for run in new_runs if run.dagster_run.tags.get(BACKFILL_ID_TAG) is None])
+# def _fetch_runs_not_in_backfill(
```

Comment:
remove... let's rely on commit history to keep track of deprecated implementation rather than comments in the repo.

Reply:
oops, this was unintentional. thanks for catching, i thought i had deleted it